tests/forte/data/ontology/test_specs/test_invalid_ndarray_shape.json

```diff
@@ -0,0 +1,18 @@
+{ 
+    "name": "NdArrayEntry", 
+    "definitions": [ 
+      { 
+        "entry_name": "ft.onto.test_ndarray.BadShapeNdEntry1", 
+        "parent_entry": "forte.data.ontology.top.Generics", 
+        "description": "A N-dimensional entity having invalid shape.", 
+        "attributes": [ 
+          { 
+              "name": "value", 
+              "type": "NdArray",
+              "ndarray_dtype": "int",
+              "ndarray_shape": "[2, 2]"
```

Comment:
I am curious how is the shape invalid

Reply:
an array of integers is expected here. so it should be a `[2, 2]` instead of a string `"[2, 2]"`; likewise, an integer like `2` is also invalid here.

Reply:
oh, i see, this is a string, insteresting